install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -11,144 +11,406 @@
+...
+metadata:
+  annotations:
+    openshift.io/scc: nfs-scc <1>
+  name: nfs-pod1 <2>
+  namespace: default <3>
+...
+```
+<1> this is the name of the SCC that the pod used (in this case, a custom SCC)
+<2> this is the name of the pod
+<3> this is the name of the project
+
+*Note:* it can be inobvious which SCC was matched by a pod, so the command above can be very useful
+in understanding the UID, supplemental groups, and selinux re-labeling in a live container.
+
+There are currently six predefined SCCs, seen below:
+```
+# oc get scc
+NAME               PRIV      CAPS      HOSTDIR   SELINUX     RUNASUSER          FSGROUP    SUPGROUP    PRIORITY
+anyuid             false     []        false     MustRunAs   RunAsAny           RunAsAny   RunAsAny    10
+hostaccess         false     []        true      MustRunAs   MustRunAsRange     RunAsAny   RunAsAny    <none>
+hostmount-anyuid   false     []        true      MustRunAs   RunAsAny           RunAsAny   RunAsAny    <none>
+nonroot            false     []        false     MustRunAs   MustRunAsNonRoot   RunAsAny   RunAsAny    <none>
+privileged         true      []        true      RunAsAny    RunAsAny           RunAsAny   RunAsAny    <none>
+restricted         false     []        false     MustRunAs   MustRunAsRange     RunAsAny   RunAsAny    <none>
+```
+*Note:* Any SCC with a strategy set to _RunAsAny_ allows arbitrary values for that strategy to be
+defined in the pod spec (and/or image). When this applies to the user id (`runAsUser`) it is prudent
+to restrict access to the SCC to prevent a container from being able to run as root.
+
+Since often a pod matches the _restricted_ SCC it is worth knowing the security this entails.
+The following observations apply to the _restricted_ SCC:
+
+* it constrains user ids due to the `runAsUser` strategy set to _MustRunAsRange_. This forces user
+id validation.
+* since a range of allowable user ids is not defined in the SCC (see `oc export scc restricted`
+for more details), the namespace's `openshift.io/sa.scc.uid-range`) range will be used for range
+checking and for a default id, if needed.
+* it causes a default user id to be produced when a user id is not specified in the pod spec
+(again due to `runAsUser` set to _MustRunAsRange_).
+* requires a selinux label (`seLinuxContext` set to _MustRunAs_) which uses the namespace's
+default MCS label.
+* allows arbitrary supplemental group ids since no range checking is required. This is a result
+of both the `supplementalGroups` and `fsGroup` strategies being set to _RunAsAny_.
+* produces no default supplemental groups for the running pod, again due to _RunAsAny_ for the
+two group strategies above. Therefore, if no groups are defined in the pod spec (or in the
+image), the container(s) will have no supplemental groups predefined.
+
+Below is the _default_ namespace and a custom SCC, shown to summarize the interactions of
+the SCC and the namespace:
+```
+oc export ns default <1>
```

Comment:
Sure. What is the reason for this? From what I recall the main difference is that `oc get...` has current status info.